spec/mailers/application_mailer_spec.rb

```diff
@@ -4,7 +4,7 @@
   let(:user) { create(:user) }
   let(:email) { VerificationMailer.with(user_id: user.id).account_ownership_verification_email }
 
-  xdescribe "#set_perform_deliveries" do
+  describe "#set_perform_deliveries" do
```

Comment:
nitpick, non-blocking: I think this is a Deliverable test, not specifically an ApplicationMailer test.